dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/GenericData.java

```diff
@@ -43,7 +43,7 @@ public void setPair(int index, String key, long value) {
         pairs.add(index, kvp);
     }
 
-    public void setPair(int index, KeyValuePairList kvps) {
+    public void setPair(int index, KeyValuePair kvps) {
```

Comment:
I think this one should still be `KeyValuePairList kvps` -- it's just for setting a kvpl at a certain index.